packages/formik/src/Formik.tsx

```diff
@@ -830,12 +830,7 @@ export function useFormik<Values extends FormikValues = FormikValues>({
         }
       }
 
-      submitForm().catch(reason => {
-        console.warn(
-          `Warning: An unhandled error was caught from submitForm()`,
-          reason
-        );
-      });
+      submitForm()
```

Comment:
Wouldn't we need to return the Promise here in order to catch anything?:

https://codesandbox.io/s/formik-typescript-playground-dg32w?file=/pages/basic.tsx

Also, trailing semicolon please.